samples/CosmosDbIntegtation/ReadThroughSamples/StreamSample.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    internal class StreamSample
+    {
+        // Redis connection string and stream names stored in local.settings.json
+        public const string redisConnectionSetting = "redisConnectionString";

```

Comment:
Nit: naming scheme, not sure if it should be REDIS_CONNECTION_SETTING or RedisConnectionSetting, but I believe the current one is probably not ideal.

(for public const members)